docs/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+<Reference any related issues in the repository: e.g. fixes #123>
+
+<Enter a description of the changes proposed in the pull request:>
```

Comment:
Doesn't it pull in a description based on the commits in the pull request. I can't remember if github does this. I know that bitbucket does.

Reply:
What I'm saying is that the git commits should be the description.

Reply:
@kurtlenfesty The commits are in a separate tab on the pull request page.